frontend/src/game/DeckSettings.jsx

```diff
@@ -4,19 +4,31 @@ import PropTypes from "prop-types";
 import App from "../app";
 import {getZoneDisplayName, ZONE_MAIN, ZONE_SIDEBOARD} from "../zones";
 import {COLORS_TO_LANDS_NAME} from "../gamestate";
+import exportDeck from "../export";
 import Select from "../components/Select";
 
-const DeckSettings = () => (
-  (App.state.isGameFinished || App.state.didGameStart)
-    ? <div className='deck-settings'>
-      <LandsPanel />
-      <DownloadPanel />
-    </div>
-    : <div/>
-);
+import "./DeckSettings.scss";
+
+const DeckSettings = () => {
+  if (App.state.didGameStart || App.state.isGameFinished) {
+    return (
+      <div className='DeckSettings'>
+        <LandsPanel />
+        <ExportDeckPanel />
+
+        {
+          App.state.isGameFinished && /draft/.test(App.state.gametype)
```

Comment:
I know in the past we took a bad habit at putting business logic inside components. Do you think this condition should be handled elsewhere, like in the app object? IMO components should be the least aware of business objects, they may just know that the game can be a "endedDraft" but if this is too much to do, it's fine. The majority of components already use this kind of things.

Reply:
Yeah I feel like I copied this pattern from somewhere. 
I agree it feels like the component knows too much. If this that different than exposing redux state? Some component in the tree needs to have this passed in...